docs/contributing.md

```diff
@@ -182,7 +215,14 @@ $ poetry install
 !!!note
         The entire file name is literally `.env`
 
-5. Open the file with any text editor and write the bot token to the files in this format: `TOKEN="my_token"`.
+5. Open the file with any text editor and write the bot token and the database URL to the files in this format:
+        * `TOKEN="my_token"`.
+        * `DATABASE_URI=postgres://modmail:modmail@localhost:5432/modmail`
```

Comment:
Should have quotes.
`DATABASE_URI='postgres://modmail:modmail@localhost:5432/modmail'`